scripts/install-chef-suse.sh

```diff
@@ -575,9 +584,12 @@ if [ -n "$CROWBAR_FROM_GIT" ]; then
         add_ibs_repo http://dist.suse.de/ibs/Devel:/Cloud:/6/SLE_12_SP1/ cloud
     fi
 
+    # Be sure that systemd-logger is deinstalled
+    zypper --non-interactive remove systemd-logger
+
     # install chef and its dependencies
     zypper -n --gpg-auto-import-keys in ruby2.1-rubygem-chef-server ruby2.1-rubygem-chef \
-        rabbitmq-server couchdb ruby2.1-rubygem-activesupport
+        rabbitmq-server couchdb sqlite3 ruby2.1-rubygem-activesupport
```

Comment:
Maybe because we don't have a pattern yet and this is required by the rails app? Just my guess.